keras_hub/src/layers/modeling/rms_normalization.py

```diff
@@ -21,12 +22,13 @@ def __init__(self, input_dim):
         )
 
     def call(self, x):
-        """
-        Applies RMS normalization to the input tensor.
+        """Applies RMS normalization to the input tensor.
+
         Args:
-            x: KerasTensor. Input tensor of shape (batch_size, input_dim).
+            x: Input tensor of shape (batch_size, input_dim).
+
         Returns:
-            KerasTensor: The RMS-normalized tensor of the same shape (batch_size, input_dim),
```

Comment:
Did `Ruff` automatically removed `KerasTensor` from here or you removed it manually? 🤔

Reply:
I removed it manually while fixing line length issues. We can return either a kerastensor (symbolic tensor) or torch or jax or tf tensor. Better to just say tensor and shorter for line length.